src/components/BulkActionsToolbar/__tests__/BulkActionsToolbar.spec.tsx

```diff
@@ -21,19 +23,19 @@ describe('BulkActionsToolbar component', () => {
             label: 'Compare',
             disabled: true,
             onClick: onCompareClick,
-        },
+        } as BulkActionsToolbarAction,
         {
             label: 'Import',
             disabled: false,
             context: 'primary',
             onClick: onImportClick,
-        },
+        } as BulkActionsToolbarAction,
         {
             label: 'Delete',
             disabled: false,
             context: 'primary',
             onClick: onDeleteClick,
-        },
+        } as BulkActionsToolbarAction,
     ];
 
     beforeEach(() => {
```

Comment:
(note: this comment is for the next line, but I cannot add it there.)

same as https://github.com/textkernel/oneui/pull/949#discussion_r1178129921. I think we should improve these tests and remove such mocks.

Reply:
:+1: